packages/front-end/components/Experiment/LinkedChanges/constants.ts

```diff
@@ -12,7 +11,7 @@ export const ICON_PROPERTIES = {
   },
   redirects: {
     color: "#11B081",
-    component: PiShuffle,
+    component: FaLink,
```

Comment:
Do you know why we decided to change this logo? We originally considered the Link icon, but decided it didn't make sense for redirects. We also use this icon in other places in the app to copy a link to a page

Reply:
I don't know the specifics - I was just updating this based on Kari's new designs.